editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -790,7 +790,7 @@ void TileMapEditorTilesPlugin::forward_canvas_draw_over_viewport(Control *p_over
 							// Get the tile modulation.
 							Color modulate = tile_data->get_modulate();
 							Color self_modulate = tile_map->get_self_modulate();
-							modulate = Color(modulate.r * self_modulate.r, modulate.g * self_modulate.g, modulate.b * self_modulate.b, modulate.a * self_modulate.a);
+							modulate = modulate * self_modulate;
```

Comment:
You could even do `modulate *= self_modulate` here.

Reply:
Done.